examples/configuration.yaml

```diff
@@ -3,4 +3,4 @@ kind: Configuration
 metadata:
   name: cofiguration-aws-network
 spec:
-  package: xpkg.upbound.io/upbound/platform-aws-network:0.1.0
+  package: xpkg.upbound.io/upbound/platform-aws-network:0.7.0
```

Comment:
v0.7.0 ?

Reply:
@haarchri 
In the description of this PR 

>The assumption is that we will have/start with the current tag increment inherited from >https://github.com/upbound/platform-ref-aws/blob/main/examples/configuration.yaml

Up to discussion. We can obviously start to release everything with a totally independent version tagging schema